docs/demos/connector/neo4j/index.rst

```diff
@@ -1,27 +1,56 @@
 Neo4j
 ====================================
 
-In this folder:
-
-- :doc:`directed-graphsage-on-cora-neo4j-example` and :doc:`undirected-graphsage-on-cora-neo4j-example` provide examples of running GraphSAGE with connection to graph database.
-
-- :doc:`load-cora-into-neo4j` provides demo of loading Cora network citation dataset into neo4j database.
-
-- :doc:`cluster-gcn-on-cora-neo4j-example` provide an example of running Cluster GCN with connection to a graph database.
-
+`StellarGraph <https://github.com/stellargraph/stellargraph>`_ provides various algorithms that can be run on graphs in Neo4J. This folder contains demos of all of them to explain how to connect to Neo4J and how to run these algorithms as part of a TensorFlow Keras data science workflow.
 
 Required Installations:
 
 - **Neo4J**: `Instruction to download <https://neo4j.com/docs/operations-manual/current/installation/>`_.
 - **APOC library** plug-in: APOC provides utilities for common procedures and functions in Neo4J. `Instruction to download and install <https://neo4j.com/developer/neo4j-apoc/>`_.
 - **py2neo**: A client library and toolkit to connect to Neo4J database from within python applications. Install ``py2neo`` by using pip: ``pip install py2neo``. `documentation <https://py2neo.org/v4/>`_.
```

Comment:
We also have a `neo4j` extra, so maybe this could be added as:

```suggestion
- **py2neo**: A client library and toolkit to connect to Neo4J database from within python applications. Install ``py2neo`` by using pip: ``pip install stellargraph[neo4j]`` or ``pip install py2neo``. `documentation <https://py2neo.org/v4/>`_.
```

That said, it's currently only py2neo that's required, so maybe the complexity of the extra installation is unnecessary and it's fine to just go for the basic form. Up to you.

Reply:
Yeah I think we might as well mention that option here